package.json

```diff
@@ -1,54 +1,29 @@
 {
   "name": "hubot",
   "version": "2.19.0",
-  "author": "hubot",
   "keywords": [
     "github",
     "hubot",
     "campfire",
     "bot"
   ],
   "description": "A simple helpful robot for your Company",
-  "license": "MIT",
+  "license": "Apache-2.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/github/hubot.git"
   },
   "dependencies": {
     "hubot": "^2.19.0",
-    "async": ">=0.1.0 <1.0.0",
-    "chalk": "^1.0.0",
-    "cline": "^0.8.2",
-    "coffee-script": "1.6.3",
```

Comment:
We still have a bunch of coffeescript files in `src/`. It doesn't look like we're using them, because we now depend on hubot rather than using the source locally, and also because of this line where we've removed the coffeescript compiler completely. What's up with that? Should they be gone too?